src/Groups/matrices/forms.jl

```diff
@@ -265,9 +265,18 @@ Given a quadratic form `Q`, return the symmetric form with Gram matrix `B`
 defined by `B(u,v) = Q(u+v)-Q(u)-Q(v)`.
 
 # Examples
-```@repl
-Q = quadratic_form(invariant_quadratic_form(GO(3,3)))
-corresponding_bilinear_form(Q)
+```jldoctest
```

Comment:
```suggestion
```jldoctest; filter = r"\[.*\]"
```
this filter removes all matrix entries. Thus, the jldoctest will not fail when entries change

Reply:
This proposal makes sense: We cannot check what the matrix contains but we can that something of the right type is returned.